src/main/java/uk/gov/hmcts/divorce/divorcecase/tab/ApplicationHistoryTab.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.divorce.divorcecase.tab;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.api.CCDConfig;
+import uk.gov.hmcts.ccd.sdk.api.ConfigBuilder;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.State;
+import uk.gov.hmcts.divorce.divorcecase.model.UserRole;
+
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.APPLICANT_1_SOLICITOR;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.APPLICANT_2_SOLICITOR;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.CASE_WORKER;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.JUDGE;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.LEGAL_ADVISOR;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.SUPER_USER;
+
+@Component
+public class ApplicationHistoryTab implements CCDConfig<CaseData, State, UserRole> {
```

Comment:
Nitpick: Event history or case history might be more consistent with the naming we use in conversation with service?
```suggestion
public class EventHistoryTab implements CCDConfig<CaseData, State, UserRole> {
```

Reply:
Had to name it so since the ccd config generator would otherwise pick up the ApplicationTab.java first and CaseTypeTab next and assign TabOrder of lower values to the tabs in these files. This means that History tab would be last.
The only other way around it to force a sequence in which ccd config generator would pickup files in this directory.

Reply:
Fiel renamed to ApplicationEventHistory.java